HyperVExtension/src/HyperVExtension/Strings/en-US/Resources.resw

```diff
@@ -337,4 +337,24 @@
     <value>Maximum attempts reached</value>
     <comment>Error description text shown in the dialog asking to log in to Hyper-V VM after the last attempt.</comment>
   </data>
+  <data name="UserNotInHyperVAdminGroup" xml:space="preserve">
+    <value>User {0} is not a member of the Hyper-V Administrators group. If you have already added the user to the group you may need to log off and back on again.</value>
+    <comment>Locked="{Hyper-V}" Error text for when the user is not in the Hyper-V adminstrators group. {0} is the username of the user</comment>
+  </data>
+  <data name="HyperVFeatureNotPresent" xml:space="preserve">
+    <value>The Hyper-V Feature is not present on this SKU of Windows.</value>
+    <comment>Locked="{Hyper-V}" Error text to show the user that the Hyper-V feature is not available on the current SKU of Windows</comment>
+  </data>
+  <data name="HyperVFeatureDisabled" xml:space="preserve">

```

Comment:
Should this and next three strings end in a ". "? (I know you're going to get the content writer to approve, but in the meantime)

Reply:
sure I can add it